src/components/UserInfo.vue

```diff
@@ -56,31 +56,38 @@
             v-on:change="getUserClientRoles()"
           ></v-autocomplete>
         </v-col>
-        <v-col class="col-7">
-          <label v-show="selectedClientId">Roles</label>
-          <div class="checkbox-group" v-if="selectedClientId">
-            <v-checkbox
-              hide-details="auto"
-              v-for="role in effectiveClientRoles"
-              v-model="selectedRoles"
-              :value="role"
-              :label="role.name"
-              v-bind:key="role.name"
-            ></v-checkbox>
-            <v-checkbox
-              hide-details="auto"
-              v-for="role in availableClientRoles"
-              v-model="selectedRoles"
-              :value="role"
-              :label="role.name"
-              v-bind:key="role.name"
-            ></v-checkbox>
-          </div>
+      </v-row>
+      
+        <div v-if="selectedClientId">
+          <v-row no-gutters>
+            <v-col class="col-4">
+              <label>Roles</label>
+              <v-checkbox
+                hide-details="auto"
+                v-for="role in clientRoles"
+                v-model="selectedRoles"
+                :value="role"
+                :label="role.name"
+                v-bind:key="role.name"
+              ></v-checkbox>
+            </v-col>
+            <v-col class="col-4">
+              <label>Effective Roles</label>
+              <v-checkbox
+                hide-details="auto"
+                v-for="role in effectiveClientRoles"
+                v-model="effectiveClientRoles"
+                disabled="disabled"
+                :value="role"
+                :label="role.name"
+                v-bind:key="role.name"
+              ></v-checkbox>
+            </v-col>
+          </v-row>
           <div class="my-6" v-if="selectedClientId">
             <v-btn class="secondary" medium v-on:click="updateUserClientRoles()">Save User Role</v-btn>
```

Comment:
Just an idea, but it might be clearer if this were "Save Permissions" or the section was called "Roles" instead of "Permissions".

![image](https://user-images.githubusercontent.com/1767127/84714512-bba31f00-af22-11ea-8f51-cabb7c5ae194.png)